docs/sql-keywords.md

```diff
@@ -179,6 +179,7 @@ Below is a list of all the keywords in Spark SQL.
   <tr><td>MONTH</td><td>reserved</td><td>non-reserved</td><td>reserved</td></tr>
   <tr><td>MONTHS</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
   <tr><td>MSCK</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
+  <tr><td>NAMESPACES</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
```

Comment:
cc @xianyinxin , we should also add DELETE and UPDATE. Can you open a PR to do it?

Reply:
DELETE is already there. UPDATE is included in https://github.com/apache/spark/pull/25626